docs/book/src/topics/dual-stack.md

```diff
@@ -0,0 +1,67 @@
+# Dual-stack clusters
+
+## Overview
+
+CAPZ enables you to create [dual-stack](https://github.com/Azure/aks-engine/tree/master/examples/dualstack) Kubernetes cluster on Microsoft Azure.
+
+To deploy a cluster using dual-stack, use the [dual-stack flavor template](https://raw.githubusercontent.com/kubernetes-sigs/cluster-api-provider-azure/main/templates/cluster-template-dual-stack.yaml).
```

Comment:
this is going to fail the lint check because that file doesn't exist in main yet